### ISSUE TITLE
Value of USE TEMPLATE in admin is wrong, but template is used

### ISSUE THREAD

**Issue by Schwankenson** [ui/django-post_office#395]
I created a email template in database admin. And the sent mail with this command:

            mail.send(
                'recipient@example.com', # List of email addresses also accepted
                'from@example.com',
                template='user_cancelled_subscription', # Could be an EmailTemplate instance or name
                context={'foo': 'bar'},
            )

Email subject and content is filled with values from the template, everything seems to work prefectly. But it admin, there is a red cross below USE TEMPLATE:
![image](https://user-images.githubusercontent.com/803955/159487916-b6308d0c-dac6-489c-b3d9-43d37e34fe5f.png)

And when I click filter `user_cancelled_subscription` for the "By Name" filter, the email is not shown.

What could be wrong? This is really confusing.

**Comment by Schwankenson** [comment]
Another test:

    mail_template = EmailTemplate.objects.get(name="user_cancelled_subscription")

    email = mail.send(
        'recipient@example.com', # List of email addresses also accepted
        'from@example.com',
        template=mail_template, # Could be an EmailTemplate instance or name
        context={'foo': 'bar'},
    )
    
    email.template = mail_template
    email.save()

Like this, the template is shown in admin and filter works. But the template variables are not filled anymore.

![image](https://user-images.githubusercontent.com/803955/159504172-1a106915-2a2a-4640-be39-00699ae91f84.png)

**Comment by selwin** [comment]
Yeah this seems to be a bug. Mind opening a PR for this? If `template` is passed to `mail.send()`, `use_template` should be `True`.

**Comment by Schwankenson** [comment]
I don`t have the time to fix this and open a PR at the moment...somebody else? Should be easy to fix, right?
